packages/elements/src/components/app/app.component.ts

```diff
@@ -88,7 +87,10 @@ export class MutationTestReportAppComponent extends RealTimeElement {
     return getComputedStyle(this).getPropertyValue('--mut-body-bg');
   }
 
-  #filePickerRef = createRef<MutationTestReportFilePickerComponent>();
+  @query('mte-file-picker')
+  private declare filePicker: MutationTestReportFilePickerComponent;
```

Comment:
I didn't know you could do this :o

Reply:
https://lit.dev/docs/components/shadow-dom/#@query-@queryall-and-@queryasync-decorators

It basically expands to this:

```ts
get filePicker() {
  return this.renderRoot?.querySelector('mte-file-picker') ?? null;
}
```